homeassistant/components/dlna_dmr/strings.json

```diff
@@ -0,0 +1,44 @@
+{
+  "config": {
+    "flow_title": "{name}",
+    "step": {
+      "user": {
+        "title": "DLNA Digital Media Receiver",
+        "description": "URL to a device description XML file",
+        "data": {
+          "url": "[%key:common::config_flow::data::url%]"
+        }
+      },
+      "confirm": {
+        "description": "[%key:common::config_flow::description::confirm_setup%]"
+      }
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "could_not_connect": "Failed to connect to DLNA device",
+      "discovery_error": "Failed to discover a matching DLNA device",
+      "incomplete_config": "Configuration is missing a required variable",
+      "non_unique_id": "Multiple devices found with the same unique ID",
+      "not_dmr": "Device is not a Digital Media Receiver"
```

Comment:
Technically, DMR stands for Digital Media Renderer, not receiver.

Reply:
In fact, home assistant becomes a DMC (Digital Media Controller), so perhaps the integration is named wrong? Though a bit late to change things...

Reply:
Whoops, fixed now. I think the integration is still named correctly, since most integrations are named for what they control and not what they are.